src/makeCalculator.js

```diff
@@ -37,7 +37,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+
+  const calculator = {
+
+    result: 0,
+
+    operate(callback, number) {
+      callback.call(this, number);
+      return this;
+    },
+
+    add(x) {
+      return this.result = this.result + x;
+
+    },
```

Comment:
shorter version to improve readability :) remember to change all cases
```suggestion
    add(x) {
      return this.result += x;
    },
```